### PR TITLE
Use I18n pluralization for a better message for new ownership requests

### PR DESCRIPTION
**Pull request by coorasse**
This adds two more translation using I18n.pluralization feature

### CODE REVIEW

**Comment by simi** [comment]
Hello and thanks for the contribution @coorasse. Would you mind to run `bin/fill-locales` script to add those to other translations? I can do that as well if needed.

**Comment by coorasse** [comment]
Thank you for the quick reply. I actually got an error when running `bin/fill-locales`:

```
undefined method `dig' for an instance of String (NoMethodError)
[key, fill(en[key], other&.dig(key))]
                               ^^^^^
	from bin/fill-locales:12:in `each'
```

I did not correct this problem, but I added them manually. I also took the chance to correct a typo in the key

**Comment by coorasse** [comment]
I am now wondering: will this create an issue? If there's only one request, but in Spanish we have an empty translation for "one", will it fallback to "other" or show an empty string? I believe it might be the second one, so we might have a problem here

**Comment by coorasse** [comment]
Ok, I did not expect this:

CleanShot 2024-02-17 at 21 43 26@2x

**Comment by simi** [comment]
@coorasse the easiest seems to find out what to put for Spanish pluralization in there. I have asked for help on bundler Slack, let's hope someone will help us to translate that.

File: app/views/owners_mailer/new_ownership_requests.html.erb [hunk 1]
```diff
@@ -34,7 +34,7 @@
                       </td>
                       <td bgcolor="#e9573f">
                         <div class="text-btn" style="color:#ffffff; font-family:Arial, sans-serif; min-width:auto !important; font-size:16px; line-height:20px; text-align:center">
-                          <a href=<%= rubygem_adoptions_url(@rubygem) %> target="_blank" class="link-white" style="color:#ffffff; text-decoration:none"><span class="link-white" style="color:#ffffff; text-decoration:none"><%= t("mailer.new_opwnership_requests.button") %></span></a>
+                          <a href=<%= rubygem_adoptions_url(@rubygem) %> target="_blank" class="link-white" style="color:#ffffff; text-decoration:none"><span class="link-white" style="color:#ffffff; text-decoration:none"><%= t("mailer.new_ownership_requests.button") %></span></a>
```
[the remark this answers is not in the document]
Comment (martinemde) [reply app/views/owners_mailer/new_ownership_requests.html.erb:37]: Good to know, but not relevant here since the link is not external.

**Comment by coorasse** [comment]
I can also translate to spanish, and also German and French, but we'll have the same issue with Japanese.
I can run those through deepl pro, if that's ok. 
Personal opinion: 
* I believe it's ok to have an English fallback
* I believe we should not be stuck by other languages translations, otherwise the barrier to contribute is too high. I think sticking with english should be enough

Let me know if it's ok to translate Spanish, french, german, and let deepl do the rest

File: app/views/owners_mailer/ownership_request_approved.html.erb [hunk 2]
```diff
@@ -26,7 +26,7 @@
                         </td>
                         <td bgcolor="#e9573f">
                           <div class="text-btn" style="color:#ffffff; font-family:Arial, sans-serif; min-width:auto !important; font-size:16px; line-height:20px; text-align:center">
-                            <a href=<%= rubygem_owners_url(@rubygem.slug) %> target="_blank" class="link-white" style="color:#ffffff; text-decoration:none"><span class="link-white" style="color:#ffffff; text-decoration:none"><%= t("mailer.new_opwnership_requests.owners_page") %></span></a>
+                            <a href=<%= rubygem_owners_url(@rubygem.slug) %> target="_blank" class="link-white" style="color:#ffffff; text-decoration:none"><span class="link-white" style="color:#ffffff; text-decoration:none"><%= t("mailer.new_ownership_requests.owners_page") %></span></a>
```
[the remark this answers is not in the document]
Comment (martinemde) [reply app/views/owners_mailer/ownership_request_approved.html.erb:29]: I opened #4463 to address this so it doesn't keep making noise.

**Comment by simi** [comment]
@coorasse I'm happy with English fallback. Let's leave it as is.

**Review by simi — APPROVED** [review]

**Review by martinemde — APPROVED** [review]
I guess it's the end of us pwning our locales. Thanks for the fix and I'm happy with the approach.

**Comment by martinemde** [comment]
P.S. I'd love to collect together known translators interested in helping translate rubygems.org (maybe it a slack channel on the Bundler slack or a discussion thread in rubygems.org). We add new untranslated text at a surprisingly regular cadence and leaving it all untranslated is unfortunate.

Please join the Bundler slack for now if you're not already and we'll work on making that happen.

**Pull request merged**